packages/wallet/src/services/types.ts

```diff
@@ -64,6 +64,7 @@ export interface OutgoingTx {
   cbor: TxCBOR;
   body: Cardano.TxBody;
   id: Cardano.TransactionId;
+  context?: HandleResolution[] | [];
```

Comment:
Same as
```suggestion
  context?: HandleResolution[];
```

Reply:
2d88f9ab41c0bf941b8126409e8d597fa2713303  should address this